CourseGrab/Network/Endpoints.swift

```diff
@@ -15,7 +15,7 @@ extension Endpoint {
         Endpoint.config.keyDecodingStrategy = .convertFromSnakeCase
         Endpoint.config.keyEncodingStrategy = .convertToSnakeCase
 
-        #if LOCAL
+        #if DEBUG
```

Comment:
Should we change this back to `#if LOCAL` or revert it back once backend deploys?

Reply:
oh I'm not so sure what's the difference between LOCAL and DEBUG lol but I'll put it back to LOCAL

Reply:
`LOCAL`/`DEBUG` refer to the build configuration that's being used when the project runs! we don't actually have a build configuration named `Local` (you can see them in when you click on just the project in Xcode sidebar in the "Info" tab), so the `#if LOCAL` is never actually hit lol -- if backend isn't deployed yet i think it's okay to keep it as `DEBUG` if you guys plan on continuing to use locally deployed backend to dev on